tickets/k8s/CoreV1NodeStatus.org

```diff
@@ -4,11 +4,12 @@
 #+TODO: TODO(t) NEXT(n) IN-PROGRESS(i) BLOCKED(b) | DONE(d)
 #+OPTIONS: toc:nil tags:nil todo:nil
 #+EXPORT_SELECT_TAGS: export
-* TODO Progress [0/4] :export:
-- [ ] mock ticket
-- [ ] issue created
-- [ ] PR created
-- [ ] promotion created
+* TODO Progress [2/5]                                                :export:
+- [X] apisnoop pr : cncf/apisnoop#317
```

Comment:
These links won't render correctly. Let's use org-links... they'll work in mark-down as well.

Reply:
Thanks @hh I resolved link issue